include/seqan3/search/fm_index/fm_index.hpp

```diff
@@ -48,7 +48,7 @@ class bi_fm_index_cursor;
  *
  * ### Running time / Space consumption
  *
- * \f$SAMPLING\_RATE = 16\f$
+ * \f$SAMPLING\_RATE = 16\f$ \n
```

Comment:
is this adding a line break in doxygen ? I've never seen this before

Reply:
@smehringer Yes, it puts the sigma on the next line.

Reply:
Alright